<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Plot</class>
 <widget class="QMainWindow" name="Plot">
  <property name="windowModality">
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>870</width>
    <height>970</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>TATSSI time series generator - Spatial Subset</string>
  </property>
  <widget class="QWidget" name="content_plot">
   <widget class="QPushButton" name="pbGetExtent">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>401</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Get extent for subsetting</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pbOverlay">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>10</y>
      <width>441</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Overlay geometry</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
